.github/workflows/widget-ntopng-listing.yml

```diff
@@ -13,25 +13,32 @@ on:
       - ".github/workflows/centreon-widget-ntopng-listing.yml"
       - "widgets/centreon-widget-ntopng-listing/**"
     tags:
-      - centreon-widget-ntopng-listing-22.10.**
-
-env:
-  version: "22.10"
+      - centreon-widget-ntopng-listing-*
```

Comment:
Why not use a single workflow using a matrix of widgets ?

Reply:
Because it needs complex configuration to run only needed widget job. It's possible but can be managed later